package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@replit/codemirror-indentation-markers",
   "description": "Render indentation markers in CodeMirror",
-  "version": "6.5.0",
+  "version": "6.5.1",
   "author": {
```

Comment:
did you already publish this? can you bump in a separate commit please?

Reply:
nope i didn't publish yet